core/frontend/src/views/ExtensionView.vue

```diff
@@ -4,18 +4,25 @@
     :source="service_path"
     :class="fullpage ? 'fullpage' : ''"
   />
+  <SpinningLogo v-else-if="detected_port === undefined" />
```

Comment:
can we put v-else-if and v-else in the beggning of the component for better readability ?

Like:
`<v-else page-not-found/>`

Reply:
that doesn't seem to be valid syntax